library/src/main/java/com/yausername/youtubedl_android/YoutubeDL.kt

```diff
@@ -161,7 +156,8 @@ object YoutubeDL {
     fun execute(
         request: YoutubeDLRequest,
         processId: String? = null,
-        callback: ((Float, Long, String) -> Unit)? = null
+        callback: ((Float, Long, String) -> Unit)? = null,
+        redirectErrorStream: Boolean = false
```

Comment:
this is a change which would break binary compatibility (especially the use of trailing lambda), could you please add a new overload for this?

Reply:
A new function? sure

Reply:
I added the meat of the function into a private function where both executes call from. Sounds good?